images/kubekins-e2e/variants.yaml

```diff
@@ -3,7 +3,7 @@ variants:
     CONFIG: experimental
     GO_VERSION: 1.12.1
     K8S_RELEASE: stable
-    BAZEL_VERSION: 0.26.0
+    BAZEL_VERSION: 0.28.0
```

Comment:
Pretty sure we know this will be badly broken (took quite a bit of work to get test-infra to work on 0.27, for example). I am quite confident bazel 0.27 will not build k/k. What's the motivation here?

Reply:
Bryce's team need a different bazel version for their e2e test (with their own build stuff, nothing to do with K8s)

I thought we always want to keep the experimental version up-to-date?

@brycecr maybe you can try to use the upstream bazel image? I forgot if there's anything from kubekins image you are relying on.

Reply:
I have gotten around dependence on the local bazel version and kubekins version by pinning the version in my repo (dowloaded by a make target from a specific bazel release), so I am not blocked on this bump.

Last I looked, it was my impression that kubekins tracked the bazel versions somewhat more closely, but I understand that may not be true, and there may well be no contract there anyway.

Reply:
Can you use an official bazel image like we do in test-infra? https://github.com/kubernetes/test-infra/blob/66d4b6e62d902405bf5bedaae9c30d0420e27abb/config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml#L43

This makes it obvious and straight-forward to change versions.